frontend/src/components/Post/Post.js

```diff
@@ -1,16 +1,18 @@
 import '../../config';
 import axios from "axios";
 import { Link } from "react-router-dom";
-import React, { useState, useEffect } from 'react'
-
-import timeElapsedSincePosted from "../../services/TimeService";
+import React, { useState, useEffect } from 'react';
 
+// Project
```

Comment:
To bring back a previous comments, though these comments are great, I feel anyone reading the Project and Stylesheet imports will know what they refer to without the "introductory" comment